### DISCUSSION TITLE
main.c

### DISCUSSION THREAD

**Discussion by SantiagoOsorio3** [IIC2133-PUC/2024-2#77] [Tarea 1] [answered]
Hola, queria pregunar si es que el codigo que viene en el main en la t1 esta incompleto. Incompleto me refiero a que falten por ejemplo el numero de S y K. Ya que por lo menos a mi solo me aparece   (int E; // Cantidad de eventos.)

En donde en este caso deberiamos ponerlo nosotros cierto ? o se debe hacer algo para solo trabajar con el (int E; // Cantidad de eventos.)

**Comment by SantiagoOsorio3** [comment]
Solucionado !

**Comment by 4n1t4** [comment, marked as answer]
Hola , las bases cada vez tendrán menos código para que vayan soltando la mano. Puedes usar cm referencia la tarea pasado para aprender a hacer los scanf y lectura de archivos. Asique si, deben completar con lo necesario para completar su tarea.
Saludos!

**Reply by SantiagoOsorio3** [reply]
Sisi, me habia confundido yo.. Gracias igual !
